Formula/forego.rb

```diff
@@ -1,8 +1,9 @@
 class Forego < Formula
   desc "Foreman in Go"
   homepage "https://github.com/ddollar/forego"
-  url "https://github.com/ddollar/forego/archive/v0.16.1.tar.gz"
-  sha256 "d4c8305262ac18c7e51d9d8028827f83b37fb3f9373d304686d084d68033ac6d"
+  url "https://github.com/ddollar/forego/archive/20170327195458.tar.gz"
+  sha256 "8a9bdff7f3d62ba64e9a8bb7fe619223e2fb20af6aff18f618ff2703a5bf1860"
+  version_scheme 1
```

Comment:
Fixed in https://github.com/Homebrew/homebrew-core/pull/14154/commits/0b154fff0ffab8c7484c676a9722e2e39746d7cd